### DISCUSSION TITLE
How to get the text of a selected table row?

### DISCUSSION THREAD

**Discussion by Ivo6** [beeware/toga#1788] [Q&A]
For a Table widget, how can I get the text of a selected row, or the text of a specific cell of a selected row?

I've tried using a function:
```
def row_sel(self, widget, **kwargs):
    if self.table.selection:
        print(self.table.selection)
```
but it returns an object:
```
<toga.sources.list_source.Row object at 0x7f4ab69b5af0>
```
I don't know any methods to access Row objects, and couldn't find any in the documentation or examples.

**Comment by mhsmith** [comment]
The relevant documentation page is [this one](https://toga.readthedocs.io/en/stable/background/data-sources.html), which says "The attributes of each Row object are the values that should be displayed." This means it has attributes that correspond to the table heading names, e.g. if your Table had `headings=["alpha", "bravo"]`, then you can do `table.selection.alpha`.

I agree this page could be a bit clearer, and it should also be linked from the main `Table` page.
